### PR TITLE
Allow easily loading individual models 

### PR DESCRIPTION
**Pull request by IgnacioJPickering**
This PR builds on my previous PR and allows easily loading single models without loading the 
whole network using 
``single_model = ANI1x(model_index=0)``
``ensemble = ANI1x()``
 for example.

### CODE REVIEW

File: tests/test_ase.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ def get_numeric_force(atoms, eps):
 class TestASE(unittest.TestCase):
 
     def setUp(self):
-        self.model = torchani.models.ANI1x().double()[0]
```
Comment (zasdfgbnm) [inline tests/test_ase.py:28]: Is the user still allowed to do `torchani.models.ANI1x().double()[0]`, if they want, as before?

Comment (IgnacioJPickering) [reply tests/test_ase.py:28]: Yes I think so, it should work same as before

Comment (IgnacioJPickering) [reply tests/test_ase.py:28]: Looks like there is a slight difference between both ways of doing this for some reason, I'll look into it

Comment (IgnacioJPickering) [reply tests/test_ase.py:28]: There is no difference, you can do both, but for some reason assertEqual() says thay are different, but it says they are different also for 2 instances of ani1x, no idea why it behaves like that.

**Comment by IgnacioJPickering** [comment]
Ok, nice, it doesn't really add anything new at all but I think the functionality is pretty obvious and should be implemented for convenience.

**Pull request merged**